.github/workflows/_sitrep_mgmn.yaml

```diff
@@ -0,0 +1,134 @@
+name: ~Generate sitrep for Multi-Node Multi-GPU tests
+
+on:
+  workflow_call:
+    inputs:
+      BADGE_FILENAME:
+        type: string
+        description: 'Name of the endpoint JSON file for shields.io badge'
+        required: true
+      ARTIFACT_NAME:
+        type: string
+        description: 'Name of the artifact zip file'
+        required: true
+      FW_NAME:
+        type: string
+        description: 'Name of the framework being used'
+        required: true
+    outputs:
+      STATUS:
+        description: 'Summary of all tests run for the workflow. Set to "success" when all metrics per job and all jobs pass, whereas a single metric failure or job error sets the status to "failure"'
+        value: ${{ jobs.sitrep.outputs.STATUS }}
+
+jobs:
+  sitrep:
+    runs-on: ubuntu-22.04
+    outputs:
+      STATUS: ${{ steps.gen-sitrep.outputs.STATUS }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+
+      - name: Download all artifacts from the previous jobs
+        uses: actions/download-artifact@v3
+
+      - name: Write exit status summary
+        id: exit-status
+        shell: bash -x -e {0}
+        run: |
+          EXIT_STATUSES="${{ inputs.FW_NAME }}-${GITHUB_RUN_ID}-*/*-status.json"
+          EXIT_STATUS_SUMMARY_FILE="exit_status_summary.json"
+          echo -e "\n\n## ${{ inputs.FW_NAME }} MGMN+SPMD Test Status" >> $EXIT_STATUS_SUMMARY_FILE
+          cat <<EOF >>$EXIT_STATUS_SUMMARY_FILE
+          | Test Case | State | Exit Code |
+          | --- | --- | --- |
+          EOF
+
+          for i in $EXIT_STATUSES; do
+            # Files are named <FW_NAME>-<GHID>-<NAME>/<NAME>-status.json
+            echo "| $(echo $i | cut -d/ -f1 | cut -d- -f2) | $(jq -r .state $i) | $(jq -r .exitcode $i)"
```

Comment:
I think `cut -d- -f2` should be `cut -d- -f3` since we want to extract `<NAME>`.  Also, this will fail for `FW_NAME`s that include dashes, right? E.g. if we want to differentiate between the pax tests and rosetta pax tests, we cannot set `FW_NAME=rosetta-pax` for the rosetta tests. Perhaps it would be better to extract the last field rather than the third to handle such cases.

Reply:
Makes sense, I will change to use last value

Reply:
Fixed in https://github.com/NVIDIA/JAX-Toolbox/pull/401/commits/827a6ce885c536737a6881f3fddf480bb0ea4e27